.gitignore

```diff
@@ -8,9 +8,6 @@ NOTES.md
 test/play.js
```

Comment:
remove  : 
 test/play.js
tmp/
installed_contracts/

why we need that here ?

Reply:
I don't know, it wasn't in the scope of this

Reply:
but I don't see a reason for them other than maybe other devs want to keep them out

Reply:
I think it is historical remaining .lets remove that.

Reply:
done